serde/src/de/impls.rs

```diff
@@ -141,12 +192,14 @@ macro_rules! impl_deserialize_num {
         visit_integer_method!(i32, visit_i32, from_i32, Signed, i64);
         visit_integer_method!(i64, visit_i64, from_i64, Signed, i64);
 
-        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64);
+        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64) ;
```

Comment:
Nitpick: looks like there's an accidental extra space here

```suggestion
        visit_integer_method!(u8, visit_u8, from_u8, Unsigned, u64);
```

Reply:
(But I guess rustfmt will get rid of all the extra spaces, so it might not really matter.)

Reply:
thank you!